src/vsr/replica.zig

```diff
@@ -5109,6 +5109,11 @@ pub fn ReplicaType(
                     if (self.op_checkpoint() == self.op) {
                         // Don't allow "op_repair_min > op_head".
```

Comment:
This feels a bit odd --- after state sync, while we are in a recovering head state (op < op_checkpoint), we  do violate this invariant.